Makefile

```diff
@@ -293,7 +293,7 @@ deploy-onprem-for-subsystem:
 	export DUMMY_IGNITION="true" && $(MAKE) deploy-onprem
 
 deploy-on-openshift-ci:
-	ln -s $(shell which oc) $(shell dirname $(shell which oc))/kubectl
+	ln -s $(shell which oc) $(shell dirname $(shell which oc))/kubectl || true
```

Comment:
Maybe `ln -f -s ...` instead?

Reply:
Not sure what the original intention was but yeah, that might make more sense.